### DISCUSSION TITLE
Can‘t write firebase.uid to my Prisma db

### DISCUSSION THREAD

**Discussion by Lil-Youn** [prisma/prisma#18541] [Q&A]
I try to write some data to my db. 

4 entrys are working as They should but the firebase.uid is not showing up in my Prisma table. 

On thé Network request the uid is shown and submitted but in my Prisma table i only get a null / empty field. Thé other 4 fields are showing thé input i submitted as They should 

Is this a known Problem or has anybody an idea Why it is like this? 

Thx in advance for your answers 🙏🏻

**Comment by ludralph** [comment]
Hi @Lil-Youn 👋 

Can you please share the relevant part of your schema file and the query you are performing?

**Reply by Lil-Youn** [reply]
Hi @ludralph this is my First question in Github. Sorry For thé missing information. Will provide more Information in around 8 hours when i am back from work!

**Reply by Lil-Youn** [reply]
don't know what happened over night but now it is working as expected.thx for your respone mate

**Reply by ludralph** [reply]
Thank you for confirming. I've gone ahead to close this issue.
